lib/render/HelloRenderer.cpp

```diff
@@ -137,20 +137,20 @@ int HelloRenderer::_paintGL(bool)
     float            fcolor[3];
     rParams->GetConstantColor(fcolor);
     if (nLights == 0) {
-        glDisable(GL_LIGHTING);
+//        glDisable(GL_LIGHTING);
```

Comment:
Would now be a good time to delete HelloRenderer.cpp/.h?

Reply:
I think that is outside the scope of this PR.